sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/ControlFlowGraph/UcfgInstructionFactory.cs

```diff
@@ -430,9 +451,8 @@ private IEnumerable<Instruction> ProcessAssignmentExpression(AssignmentExpressio
                     break;
 
                 case UcfgExpression.ElementAccessExpression elementExpression:
-                    instructions.AddRange(
-                        CreateArraySetCall(assignmentExpression, elementExpression.TypeSymbol, elementExpression.Target,
-                            rightExpression));
+                    instructions.AddRange(CreateArraySetCall(assignmentExpression, elementExpression.TypeSymbol,
```

Comment:
Other places were not splitting the method call on a new line after the `AddRange`.